libs/service-portal/signature-collection/src/screens/Parliamentary/OwnerView/ViewList/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Box, Stack, Text } from '@island.is/island-ui/core'
+import { useLocale, useNamespaces } from '@island.is/localization'
+import { m } from '../../../../lib/messages'
+import { useLocation } from 'react-router-dom'
+import { useGetSignatureList } from '../../../../hooks'
+import format from 'date-fns/format'
+import Signees from './Signees'
+import CancelCollection from '../../../Presidential/OwnerView/CancelCollection'
+
+const ViewList = () => {
+  useNamespaces('sp.signatureCollection')
+  const { formatMessage } = useLocale()
+  const { pathname } = useLocation()
+  const listId = pathname.replace(
+    '/min-gogn/listar/althingis-medmaelasofnun/',
```

Comment:
Is there a link resolver available for the service portal?

Furthermore, isn't this path defined in [libs/service-portal/signature-collection/src/lib/paths.ts](https://github.com/island-is/island.is/pull/15451/files#diff-98fcc56bd90dbc777d26effd8f556b935150b94f170602e2d18b9a9d3982cdda)